docs/config/global.md

```diff
@@ -2,6 +2,8 @@
 
 All keys from the [Common Config](common.md) are valid in addition to the keys documented here.
 
+Note that to make use of the Vault credentials (ie `credential: !!com.boxboat.jenkins.library.credentials.vault.*`), [dockcmd](https://github.com/boxboat/dockcmd) must be installed on the jenkins agents. This functionality also requires the mask-passwords plugin for Jenkins to be installed.
```

Comment:
I think the documentation would be clearer if Vault Credentials are not inermixed with existing documentation, and instead had it's own documentation page.  This way all of the different Vault Credentials types can be referenced in the same place

Reply:
Move to its own docs